pal/audio/minigame/player-minigame.ts

```diff
@@ -107,6 +107,9 @@ export class AudioPlayerMinigame implements OperationQueueable {
         this._cacheTime = 0;
         this._needSeek = false;
         this._seeking = false;
+        if ((HUAWEI || VIVO || OPPO) && this._innerAudioContext) {
```

Comment:
Only these platforms have problem?

Reply:
I have tested Taobao and Alipay platforms, and they have no problem